src/lib/apis/objkt/queries.ts

```diff
@@ -12,3 +14,23 @@ export const buildGetCollectiblesQuery = () => gql`
     }
   }
 `;
+
+const ADULT_ATTRIBUTE_NAME = '__nsfw_';
+
+export const buildGetUserAdultCollectiblesQuery = (address: string) =>
```

Comment:
Please try using query variables like here:
https://www.apollographql.com/blog/apollo-client/how-to-filter-and-search-using-variables-in-apollo-client/#refetching-and-filtering